images/linux/scripts/installers/cmake.sh

```diff
@@ -11,7 +11,7 @@ if command -v cmake; then
 else
 	json=$(curl -s "https://api.github.com/repos/Kitware/CMake/releases")
 	latest_tag=$(echo $json | jq -r '.[] | select(.prerelease==false).tag_name' | sort --unique --version-sort | tail -1)
```

Comment:
@whywaita , Could you please exclude `-rc` from the output?
`echo $json | ... | grep -v "\-rc"`

Reply:
your suggested change to avoid RC release, Is it correct?

I agree to avoid RC release. I will add `grep -v` if correct.

Reply:
Yep, that's right (avoid RC release).

Reply:
ok, I agree! I will change this.

Reply:
I changed in https://github.com/actions/virtual-environments/pull/2731/commits/448d202f9d43fe4b06326f45fa0f7f550ed68f36